### ISSUE TITLE
legacy-swc-helper uses npm aliases - not compatible with codesandbox

### ISSUE THREAD

**Issue by ling1726** [swc-project/swc#7843]
### Describe the bug

Since version 0.4.36, the dependency `legacy-swc-helper` was introduced. This is unsupported by codesandbox. This could reasonably be a missing feature for codesandbox but, just wanted to post the issue here so that the maintainers are at least aware, since it could affect other platforms that don't support npm package aliases.

### Input code

```typescript
sandbox to repro: https://codesandbox.io/s/eloquent-minsky-jlq5f9?file=/package.json
```

### Expected behavior

Running code on codesandbox that uses @swc/helpers >= 0.4.36 should not error

### Version

0.4.36

**Comment by kdy1** [comment]
I think a new version of a maintained package should not use `@swc/helpers@0.4.x`.

`v0.4.36` is a backport for next.js users who depends on an old package which depends on `@swc/helpers@0.4.x`

**Comment by ling1726** [comment]
Yeah, upgrading to 0.5.x is also on our radar. At the same time, it might not always be easy for all maintainers to roll out changes so quickly, especially to a new major version similarly to why this backport is necessary for next.js users.

I wonder why the npm alias was necessary from version 0.4.36? that alias is the main problem, does it fix anything important?. I've triggered an issue to csb since they should also legitimately support npm aliases as a feature but don't.

We have found a workaround which is to pin the version to 0.4.35 in our codesandbox templates.

**Comment by ling1726** [comment]
Sure, we can pin to 0.4.10 while we find the time to bump to 0.5.x.

**Comment by Stan-Stani** [comment]
**Workaround:** I ran into this error today and wasn't able to figure out how to use package.json's resolutions feature to force the lower version in CodeSandbox, but once I clicked `Terminal` at the lower right and used the more advanced vm feature I was able to fluent-ui to work. Seems the vm supports the dependency we're discussing here.
